api/services/gameService.js

```diff
@@ -109,7 +111,8 @@ module.exports = {
                   if (game.players.length > 1) {
                     const p0 = userService.findUser({ userId: game.players[0].id });
                     const p1 = userService.findUser({ userId: game.players[1].id });
-                    return Promise.all([Promise.resolve(game), p0, p1]);
+                    const spectatingUsers = game.spectatingUsers.map((user) => user.username);
```

Comment:
Will this always be an array?

Reply:
When there are no spectators this is an empty array in the database